### PR TITLE
Rework documentation

### PR DESCRIPTION
**Pull request by rbw**
- Update CLI description
- Update CLI usage
- Update description in readme
- Add Dimensions
- Add ECS Running Tasks example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,43 +29,45 @@ how things maps to **check_aws** input.
...
+                    {af-south-1,ap-east-1,ap-northeast-1,ap-northeast-2,ap-northeast-3,ap-south-1,ap-southeast-1,ap-southeast-2,ca-central-1,eu-central-1,eu-north-1,eu-south-1,eu-west-1,eu-west-2,eu-west-3,me-south-1,sa-east-1,us-east-1,us-east-2,us-west-1,us-west-2}
+                    [-u UNIT] -m METRIC -n NAMESPACE [-d [DIMENSIONS]]
+                    [-p PROFILE]
+                    [-s {Average,Sum,SampleCount,Maximum,Minimum}]
+                    [-w WARNING] [-c CRITICAL] [-v] [-P [PERIOD]] [-D DELTA]
+                    [-l LAG] [-C [CREDENTIALS_FILE]] [-f [CREDENTIALS_FILE]]
+
+Plugin for monitoring AWS via CloudWatch
+
+optional arguments:
```
Comment (jimorie) [inline README.md:42]: Strictly speaking `-r` is not an *optional* argument, is it?

**Review by jimorie — APPROVED** [review]
Nice improvements! :+1:

File: README.md (hunk 1 above)
Comment (rbw) [reply README.md:42]: I think you're right. Not sure why it ends up under "optional arguments" in argparse --help.

Comment (jimorie) [reply README.md:42]: 🤷  If it is like that in the CLI output, even more argument to leave it as is.

**Comment by rbw** [comment]
8f45043 was decided in TCP-4063. The "product documentation" overview has also been updated with this text.

**Pull request merged**